javascript-package/package.json

```diff
@@ -31,7 +31,7 @@
     "regenerator-runtime": "^0.13.9"
   },
   "devDependencies": {
-    "@babel/cli": "^7.17.6",
+    "@babel/cli": "^7.17.10",
```

Comment:
I closed the corresponding dependabot pr and added the change here to avoid merge conflicts.